Expand contractions and fix typos (#665)
jspecify/jspecify: docs/docs/spec.md

1.  Does `@Nullable` appear directly on that type usage?
2.  What is the [nullness operator] of that type usage?
3.  Is it reasonable to assume that `null` will not come "out" of it?

> 4.  Is it reasonable to assume that `null` can not be put "in" to it?

Reviewer: ```suggestion
4.  Is it reasonable to assume that `null` cannot be put "in" to it?
```
